community/SDK-GOVERNANCE.md

```diff
@@ -10,7 +10,42 @@ The community is organized as follows:
 
 ## New maintainers
 
-TBD
+Maintainers are a crucial part of this community. They keep our projects alive
+and ensure both development and maintenance. We define a set of criteria,
+followed by a vote, to include an active contributor as maintainer of one or
+more SDK projects. An active contributor may propose themselves as a maintainer,
+although we encourage the existing maintainers to nominate active contributors
+proactively.
+
+The criteria for a contributor to become a maintainer for a given SDK repository
+are:
+
+1. At least 20 reviews on eligible PRs
+1. At least 20 submitted eligible PRs as author
+
+When we refer to the term _eligible pull requests_, we explicit mean pull
```

Comment:
We may be able to get rid of this paragraph if we adopt what I said above. I'm not sure we should shift the definition of "substantial" to each SDK because that still implies they'll define a number, and based on last week's call I think a lot of folks are nervous about picking a number. So it might be best to leave it a bit vague.